packages/utilities/src/overflowHandler/README.md

```diff
@@ -0,0 +1,51 @@
+# overflowHandler
+
+`overflowHandler` is a framework-agnostic utility for handling the truncation of
+a list of items.
+
+## Getting started
+
+Here's a quick example to get you started.
+
+```js
+import { createOverflowHandler } from '@carbon/utilities';
+
+createOverflowHandler({
+  container, // The document node that holds the items. HTMLElement
+  maxVisibleItems, // Maximum number of visible items. If provided, only this number of items will be shown. number | undefined
+  onChange: (visibleItems: HTMLElement[], hiddenItems: HTMLElement[]) => void, // Callback function invoked when the visible and hidden items change.
+  dimension, // The dimension to consider for overflow calculations. "width" | "height" | undefined
+});
```

Comment:
Any reason that `onChange` is the only property which includes types? If not, I suggest adding types for everything else - types that match the function's interface.

```suggestion
createOverflowHandler({
  container: HTMLElement,
  maxVisibleItems?: number,
  onChange: (visibleItems: HTMLElement[], hiddenItems: HTMLElement[]) => void,
  dimension?: "width" | "height",
});
```